### ISSUE TITLE
Problems inferring BLAS info when running $ neon examples/mnist_mlp.yaml?

### ISSUE THREAD

**Issue by tianqig** [NervanaSystems/neon#345]
Hi, 

not sure what's wrong with my neon installation, got the following error messages, any one can help?
On  a MacBook Pro, OS X Sierra, Python 2.7.13 Looks it's a url loading error.

thanks
======================================================
(.venv2) MacBook:neon Allen$ neon examples/mnist_mlp.yaml
2017-01-26 21:31:50,121 - neon.backends.nervanacpu - WARNING - Problems inferring BLAS info, CPU performance may be suboptimal
Traceback (most recent call last):
  File "/Users/Allen/neon/.venv2/bin/neon", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/Users/Allen/neon/bin/neon", line 171, in <module>
    train, test = load_data(data_dir=args.data_dir, backend_obj=model.be)
  File "/Users/Allen/neon/bin/neon", line 150, in load_data
    dataiters = dataset.gen_iterators()
  File "/Users/Allen/neon/neon/data/image.py", line 78, in gen_iterators
    (X_train, y_train), (X_test, y_test), nclass = self.load_data()
  File "/Users/Allen/neon/neon/data/image.py", line 64, in load_data
    self.fetch_dataset(self.url, self.filename, filepath, self.size)
  File "/Users/Allen/neon/neon/data/datasets.py", line 134, in fetch_dataset
    cloudfile = urlopen(req)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 429, in open
    response = self._open(req, data)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 447, in _open
    '_open', req)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1241, in https_open
    context=self._context)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 1198, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661)>

**Comment by ursk** [comment]
We cannot reproduce your error on a MacBook Pro with OS X Sierra and Python 2.7.12 so this appears to be specific to your system configuration. Try checking firewall rules etc. and make sure urllib2 can connect freely. Make sure you are using a supported Python distribution, e.g. from Brew. Also see http://stackoverflow.com/questions/27835619/ssl-certificate-verify-failed-error/27847883 for some more context and possible work-arounds.

**Comment by tianqig** [comment]
Hi ursk, from your suggested url, I found an easy way to try  is to set: 
PYTHONHTTPSVERIFY=0 

I added this statement in .bash_profile , somehow my problem remained. Then I found the following  website: [Certificate verification in Python standard library HTTP clients](https://access.redhat.com/articles/2039753), then I tried
 $ PYTHONHTTPSVERIFY=0  python examples/mnist_mlp.py

No more errors for the mnist demo. Not sure why the change of my .bash_profile did not fix my problem.

thank ursk for quick reply and suggestions. BTW, I checked my urllib2 worked no problem.
